server/pages/index.js

```diff
@@ -322,6 +323,24 @@ function setUpLoggedInRoute( req, res, next ) {
 		'X-Frame-Options': 'SAMEORIGIN',
 	} );
 
+	const LANG_REVISION_FILE_URL = 'https://widgets.wp.com/languages/calypso/lang-revisions.json';
+	const langPromise = superagent
+		.get( LANG_REVISION_FILE_URL )
+		.then( response => {
+			const languageRevisions = response.body;
+
+			req.context.languageRevisions = languageRevisions;
```

Comment:
Do we need to sanitize the network response?

Reply:
Yes, we do. Thanks for catching this flaw. 🙇 
To not further bloat the size of this PR, I'm going to address this in an up-coming one.

Reply:
Here it goes: https://github.com/Automattic/wp-calypso/pull/27849